linux_os/guide/system/software/system-tools/package_tuned_removed/rule.yml

```diff
@@ -28,6 +28,11 @@ references:
 
 {{{ complete_ocil_entry_package(package="tuned") }}}
 
+{{% if product == "rhel8" %}}
+platforms:
+    - not_ovirt-host
```

Comment:
Something along: `This rule is disabled for Red Hat Virtualization, it will report not applicable. RHV requires tuned package for tuning profiles that can enhance virtualization performance.`